src/app/api/auth/login/getUserData.test.ts

```
@@ -0,0 +1,240 @@
+import { NextRequest } from 'next/server'
```

ESLint

[error] 'NextRequest' is defined but never used.

(@typescript-eslint/no-unused-vars)